include/deal.II/base/derivative_form.h

```diff
@@ -89,46 +91,48 @@ class DerivativeForm
   operator=(const Tensor<1, dim, Number> &);
 
   /**
-   * Converts a DerivativeForm <order,dim,dim> to Tensor<order+1,dim,Number>.
-   * In particular, if order==1 and the derivative is the Jacobian of F, then
-   * Tensor[i] = grad(F^i).
+   * Converts a DerivativeForm <order, dim, dim, Number> to Tensor<order+1, dim,
+   * Number>. In particular, if order == 1 and the derivative is the Jacobian of
+   * $\mathbf F(\mathbf x)$, then Tensor[i] = $\nabla F_i(\mathbf x)$.
    */
   operator Tensor<order + 1, dim, Number>() const;
 
   /**
-   * Converts a DerivativeForm <1, dim, 1> to Tensor<1,dim,Number>.
+   * Converts a DerivativeForm<1, dim, 1, Number> to Tensor<1, dim, Number>.
    */
   operator Tensor<1, dim, Number>() const;
 
   /**
-   * Return the transpose of a rectangular DerivativeForm, that is to say
+   * Return the transpose of a rectangular DerivativeForm,
    * viewed as a two dimensional matrix.
    */
   DerivativeForm<1, spacedim, dim, Number>
   transpose() const;
 
   /**
    * Compute the Frobenius norm of this form, i.e., the expression
-   * $\sqrt{\sum_{ij} |DF_{ij}|^2}$.
+   * $\sqrt{\sum_{ij} |DF_{ij}|^2} =
+   * \sqrt{\sum_{ij} |\frac{\partial F_i}{\partial x_j}|}$.
```

Comment:
You'll need the square here as well:
```suggestion
   * \sqrt{\sum_{ij} |\frac{\partial F_i}{\partial x_j}|^2}$.
```